.travis.yml

```diff
@@ -1,20 +1,15 @@
+dist: trusty
 language: java
 sudo: false
-install: mvn -Pstaging install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
-script: mvn -Pstaging test -B
+
 jdk:
-  - oraclejdk8
+  - oraclejdk11
```

Comment:
Why oracle version? Isn't openjdk good enough? From what I recall, there was absolutely no difference between those two since JDK 9.

Reply:
You're right. I'll switch to open jdk 8.

Reply:
I meant to have both - `openjdk8` and `openjdk11`

Reply:
yes, that's what I understood